sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 ### Breaking Changes
 
 - `PiiEntityDomainType` was renamed to `PiiEntityDomain`.
+- `domain` property of `RecognizePiiEntitiesAction` was renamed to `domainFilter`.
+- `categoriesFilter` was added to `RecognizePiiEntitiesAction`.
```

Comment:
do you have an issue tracking adding tests for this?

Reply:
I just opened one here: https://github.com/Azure/azure-sdk-for-js/issues/15615.